tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -279,6 +314,9 @@ final class TestingServer(
     Semanticdbs.printTextDocument(textDocument)
   }
 
+  def bufferContent(filename: String): Option[String] =
+    buffers.get(toPath(filename))
```

Comment:
Better to return `String` and on error `throw new NoSuchElementException(filename)`